Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM python:3.10
 
 WORKDIR /app
 
+ARG FASTAPI_ENV='development'
```

Comment:
Remove this, because we use this docker file in staging and prod deployments

Reply:
sure